HunterPie.UI/Overlay/Widgets/Wirebug/Views/WirebugView.xaml

```diff
@@ -16,6 +16,8 @@
         <BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
         <converters:RatioConverter x:Key="RatioConverter"/>
         <converters:ReverseRatioConverter x:Key="ReverseRatioConverter"/>
+        <!-- <converters:WirebugConditionToIconConverter x:Key="WirebugConditionToIconConverter"/> -->
+        <!-- <converters:WirebugConditionToColorConverter x:Key="WirebugConditionToColorConverter"/> -->
```

Comment:
Don't comment unused code, delete it instead.

Reply:
Deleted